content/03.biochemistry-basics.md

```diff
@@ -24,8 +24,8 @@ Post-translational modifications (PTMs) are important biological regulators cont
 Proteins can be post-translationally modified through enzymatic and non-enzymatic reactions _in vivo_ and _in vitro_ [@doi:10.1093/database/baab012]. 
 PTMs can be reversible or irreversible, and they change protein function in several ways, for example by altering substrate–enzyme interactions, subcellular localization or protein-protein interactions [@PMID:33826699; @PMID:24217768]. 
 
-More than biological 400 PTMs have been discovered in both prokaryotic and eukaryotic cells. 
-There are many more chemical artifact PTMs that occur during sample preparation, such as carbamylation. 
+More than 400 biological PTMs have been discovered in both prokaryotic and eukaryotic cells. 
+There are many more chemical artifact PTMs that occur during sample preparation, such as carbamidomethylation. 
```

Comment:
```suggestion
There are many more chemical artifact PTMs that occur during sample preparation, such as carbamylation. 
```

Reply:
I mention carbamylation (resulting from nonenzymatic reaction of isocyanic acid from urea with nucleophiles in peptides) specifically as a "chemical artifact" to indicate it is unwanted